.github/workflows/release-brew.yml

```diff
@@ -19,43 +19,35 @@ jobs:
     - name: Install dafny
       run: |
         brew install dafny
-    - name: Make test program
-      run: |
-        echo "method Main() { assert true; print 42, ['\n']; }" > a.dfy
-        echo "method m() { assert false; }" > b.dfy
-        echo "42" > expect.txt
     - name: Versions
       run: |
-        dafny -version
         which dafny
-        # -version is not supported in old dafny,
-        # but this gives the version with an error
+        dafny -version
         dotnet --list-sdks
         go version
         node --version
         java -version
-    - name: Check
-      run: dafny /compile:0 a.dfy
-    - name: Check bad
-      run: dafny /compile:0 b.dfy || echo EXPECTED FAILURE
-    ## Check that a simple program compiles and runs on each supported platform
-    - name: Check C# compile
-      run: |
-        dafny /compileVerbose:0 /compile:3 /compileTarget:cs /spillTargetCode:3 a.dfy
-    - name: Check Go compile
-      run: |
-        dafny /compile:3 /spillTargetCode:3 /compileTarget:go a.dfy
-        env GO111MODULE=auto GOPATH=$PWD/a-go go run a-go/src/a.go > actual.txt
-        diff expect.txt actual.txt
-    - name: Check Javascript compile
+        python --version
```

Comment:
Seems odd to check the Python version right before installing it (presumably installing a newer version).